AUTHORS.rst

```diff
@@ -8,3 +8,4 @@ Authors
 - Bojan Serafimov (github: bojanserafimov)
 - Vladimir Maksimovski (github: realnerom)
 - Qi Qi (github: qqi0O0)
+- Leon Wu (github: lwprogramming)
```

Comment:
Are you sure capitalization doesn't matter? Your username appears to be `LWprogramming` to me.

Reply:
Github is case-insensitive (e.g. Vlad's username)